apps/playground-api/src/PlaygroundBlock.ts

```diff
@@ -12,7 +10,9 @@ export class PlaygroundBlock {
 
   @Interval(3000)
   async generate (): Promise<void> {
-    await this.client.call('generatetoaddress', [1, PlaygroundSetup.address, 1], 'number')
+    const { masternodes }: mining.MiningInfo = await this.client.mining.getMiningInfo()
+    const randomOperator = masternodes[Math.floor(Math.random() * masternodes.length)].operator
```

Comment:
i am not expert in this, but what if we mint the block sequentially, maybe just 4 in total, not all to save time? in that way we can remove the randomness and be sure which MN to use for testing, need your thought on this @fuxingloh .

Harsh's thought is to keep this randomness to simulate the actual blockchain behaviour

Reply:
Yeah, you're right. I would say sequential is better since you can anticipate them for tests.

I am averse to Harsh idea since if we want them to be random to simulate actual blockchain behavior, we should just use TestNet, or rather just use MainNet all together.